.github/workflows/update.yml

```diff
@@ -0,0 +1,152 @@
+name: Update vuln-list
+on: workflow_dispatch
+
+jobs:
+  update:
+    name: Update repo vuln-list
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Setup github user email and name
+      run: |
+        git config --global user.email "action@github.com"
+        git config --global user.name "GitHub Action"
+        
+    - name: Run NVD
+      env:
+        VULNLIST_REPOSITORY_OWNER: ${{ secrets.Owner }}
+        GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```

Comment:
In addition, can we use env in job? I'd like to avoid writing `env` for all steps.
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idenv